lms/djangoapps/support/static/support/jsx/entitlements/data/api/client.js

```diff
@@ -25,6 +25,7 @@ const postEntitlement = ({ username, courseUuid, mode, action, comments = null }
       course_uuid: courseUuid,
       user: username,
       mode,
+      is_refundable: false,
```

Comment:
Should this still be here with the other code changes?

Reply:
This seems to be using the old name, Or am I missing something.

Reply:
no you're definitely right. Good catch